src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+/*eslint-disable*/
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  if (selectedTabId === 'asdasd') {
+    selectedTabId = tabs[0].id;
+  }
```

Comment:
let's remove this and use only `activeTab`

Reply:
But when I delete this code and add smth liket that :
```js
 const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0].id;
```
One of the tests can't pass : 
![image](https://github.com/mate-academy/react_tabs-js/assets/142230575/144df1f8-3759-4422-b85d-cf45d9948ddf)

Reply:
`className={activeTab.id === tab.id ? 'is-active' : ''}`

it should help to pass the tests